Marlin/src/feature/leds/pca9632.cpp

```diff
@@ -89,21 +92,25 @@ static void PCA9632_WriteRegister(const byte addr, const byte regadd, const byte
   Wire.endTransmission();
 }
 
-static void PCA9632_WriteAllRegisters(const byte addr, const byte regadd, const byte vr, const byte vg, const byte vb) {
+static void PCA9632_WriteAllRegisters(const byte addr, const byte regadd, const byte vr, const byte vg, const byte vb
+        #if ENABLED(PCA9632_RGBW)
+          , const byte vw
+        #endif
+                                                                                                                    ) {
   #if DISABLED(PCA9632_NO_AUTO_INC)
     uint8_t data[4], len = 4;
     data[0] = PCA9632_AUTO_IND | regadd;
     data[1 + (PCA9632_RED >> 1)] = vr;
     data[1 + (PCA9632_GRN >> 1)] = vg;
     data[1 + (PCA9632_BLU >> 1)] = vb;
   #else
-    uint8_t data[6], len = 6;
-    data[0] = regadd + (PCA9632_RED >> 1);
-    data[1] = vr;
-    data[2] = regadd + (PCA9632_GRN >> 1);
-    data[3] = vg;
-    data[4] = regadd + (PCA9632_BLU >> 1);
-    data[5] = vb;
+    PCA9632_WriteRegister(addr, regadd + (PCA9632_RED >> 1), vr);
+    PCA9632_WriteRegister(addr, regadd + (PCA9632_GRN >> 1), vg);
+    PCA9632_WriteRegister(addr, regadd + (PCA9632_BLU >> 1), vb);
+
+    #if ENABLED(PCA9632_RGBW)
+      PCA9632_WriteRegister(addr, regadd + (PCA9632_WHT >> 1), vw);
+    #endif
   #endif
   Wire.beginTransmission(I2C_ADDRESS(addr));
   Wire.write(data, len);
```

Comment:
you remove the data declaration, so this code isn't compiling at all. Can you take a look and test it?

Reply:
Ok, stupid me... I probably just did changes, compiled it, removed unnecessary data declaration and didn't test it again. Well, no just github newbie question: I can fix code but what to do next? It's possible in some way add new commit with this fix to this pull request?

Reply:
just edit, commit an push to your repo, it will be added to the PR automatically